cli/objects/jira_base.py

```diff
@@ -66,16 +74,25 @@ def create_issue(
         :param description: Description of the issue
         :param issue_type: Issue type (Bug, Task, etc.)
         :param file_attachments: An optional list of file paths. Each file in the list will be attached to the issue
+        :param labels: An optional list of labels to add to the issue
 
         :returns: A Jira Issue object
         """
-
-        issue_dict = {
-            "project": {"key": project},
-            "summary": summary,
-            "description": description,
-            "issuetype": {"name": issue_type},
-        }
+        if labels:
+            issue_dict = {
+                "project": {"key": project},
+                "summary": summary,
+                "description": description,
+                "issuetype": {"name": issue_type},
+                "labels": labels,
+            }
+        else:
+            issue_dict = {
+                "project": {"key": project},
+                "summary": summary,
+                "description": description,
+                "issuetype": {"name": issue_type},
+            }
```

Comment:
```suggestion
issue_dict = {
    "project": {"key": project},
    "summary": summary,
    "description": description,
    "issuetype": {"name": issue_type},
    "labels": labels,
}

if labels:
    issue_dict.update({"labels": labels})
```

```python
In [14]: issue_dict = {"project": {"key": "key123"}, "summary": "summary", "description": "description", "issuetype": {"name": "issue-name"}}

In [15]: labels = ""

In [16]: if labels:
    ...:     issue_dict.update({"labels": labels})
    ...: 

In [17]: issue_dict
Out[17]: 
{'project': {'key': 'key123'},
 'summary': 'summary',
 'description': 'description',
 'issuetype': {'name': 'issue-name'}}

In [18]: labels = "label-123"

In [19]: if labels:
    ...:     issue_dict.update({"labels": labels})
    ...: 

In [20]: issue_dict
Out[20]: 
{'project': {'key': 'key123'},
 'summary': 'summary',
 'description': 'description',
 'issuetype': {'name': 'issue-name'},
 'labels': 'label-123'}

In [21]:
```

Reply:
Just a suggestion to reduce replicating the initialization of `issue_dict`. Create it once and append as needed.

Reply:
@ryankwilliams thanks, the change has been made!

Reply:
@calebevans I think my indentation was off on my suggestion, github UI :)